lib/compat/wordpress-6.1/block-patterns.php

```diff
@@ -186,6 +186,12 @@ function gutenberg_register_theme_block_patterns() {
 function gutenberg_register_core_block_patterns() {
 	$should_register_core_patterns = get_theme_support( 'core-block-patterns' );
 
+	// Register categories used for block patterns.
+	$pattern_category_registry = WP_Block_Pattern_Categories_Registry::get_instance();
+	if ( ! $pattern_category_registry->is_registered( 'heading' ) ) {
+		register_block_pattern_category( 'heading', array( 'label' => __( 'Headings & Titles', 'gutenberg' ) ) );
```

Comment:
Can we make this Hero or Banner instead?

Reply:
As [Anne points out](https://github.com/WordPress/gutenberg/pull/44203#issuecomment-1248534407), "Banners" might work better across locales, and — to me — feels less like jargon than "Hero". But I don't feel strongly about this. You?